2-mandatory/1-weather-report.js

```diff
@@ -12,52 +12,51 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+  // TODO
+  let listOfSentences = [];
+  for (let i = 0; i <= cities.length; i++) {
+    listOfSentences.push(
+      `The temperature in London is ${temperatureService(cities[i])} degrees`
```

Comment:
Replace London with cities[i] so that the city name will be returned according to the temperature